tests/test_ui.py

```diff
@@ -80,14 +81,14 @@ def do_urls(env, data_element, release, expected_status=200):
 
     # if the HTML did not contain a valid JS src!
 
-
-RAW_DATA = getData(path=pytest.config.getoption("data"))
-DATA = getFlatData(RAW_DATA)
+MAIN_DATA = getMainData()
+SUPPLEMENTAL_DATA = getData(path=pytest.config.getoption("data"))
+DATA = getFlatData(MAIN_DATA, SUPPLEMENTAL_DATA)
 APP = pytest.config.getoption("app")
 
 if APP:
-    assert APP in RAW_DATA, "invalid app... you asked for {}".format(APP)
-    DATA = getFlatData({APP: RAW_DATA[APP]})
+    assert APP in SUPPLEMENTAL_DATA, "invalid app... you asked for {}".format(APP)
```

Comment:
Shouldnt this look for the key in the combined data?
Seems like the app has to exist in either (main || additional)  or (main) one of those.